packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -71,42 +71,62 @@ const ActiveVisitsTable = () => {
   const currentPathName = window.location.pathname;
   const fromPage = getOriginFromPathName(currentPathName);
 
-  const headerData = useMemo(
-    () => [
-      {
-        id: 0,
-        header: t('visitStartTime', 'Visit Time'),
-        key: 'visitStartTime',
-      },
-      {
-        id: 1,
-        header: t('idNumber', 'ID Number'),
-        key: 'idNumber',
-      },
-      {
-        id: 2,
-        header: t('name', 'Name'),
-        key: 'name',
-      },
-      {
-        id: 3,
-        header: t('gender', 'Gender'),
-        key: 'gender',
-      },
-      {
-        id: 4,
-        header: t('age', 'Age'),
-        key: 'age',
-      },
-      {
-        id: 5,
-        header: t('visitType', 'Visit Type'),
-        key: 'visitType',
-      },
-    ],
-    [t],
+  const computeHeaderData = useCallback(
+    (t, config) => {
+      const headers = [
+        {
+          id: 0,
+          header: t('visitStartTime', 'Visit Time'),
+          key: 'visitStartTime',
+        },
+      ];
+
+      config?.activeVisits?.identifiers?.map((identifier, index) => {
+        headers.push({
+          id: index + 1,
+          header: t(identifier?.header?.key, identifier?.header?.default),
```

Comment:
I don't think these translations will actually work, we should probably find strings for them.

Reply:
Hello @hadijahkyampeire, thanks for the reply.

Regarding the headers, I developed having in count that I don't know how many columns the client desires (although if no configuration is provided the previous behavior will work). 
Also, the client wanted these headers to be placed between the **_Visit Time_** and **_Name_**, thus the complexity of the headers and indexation.

I tried today the translations one more time with the Spanish file and it was working.

You can try with this config and respective keys in translation files:


```
{
  "@openmrs/esm-home-app": {
    "extensionSlots": {
      "homepage-widgets-slot": {
        "order": [
          "active-visits-widget"
        ],
        "configure": {
          "active-visits-widget": {
            "activeVisits": {
              "identifiers": [
                {
                  "header": {
                    "key": "idNumber",
                    "default": "ID Number"
                  },
                  "identifierName": "HSU ID"
                },
                {
                  "header": {
                    "key": "idLegacy",
                    "default": "ID Legacy"
                  },
                  "identifierName": "Legacy ID"
                }
              ],
              "attributes": [
                {
                  "header": {
                    "key": "telephoneNumber",
                    "default": "Telephone Number"
                  },
                  "display": "Telephone Number"
                }
              ]
            }
          }
        }
      }
    }
  }
}
```

Reply:
okay, thanks @icrc-agomes I will test it with the configurations, thanks.

Reply:
Hello @hadijahkyampeire, just a small reminder when you are able to review this PR please do it :)
Thanks a lot!

Reply:
@hadijahkyampeire do you have any update regarding this PR? thank you cc @icrc-agomes @icrc-fdeniger @denniskigen